changelog.txt

```diff
@@ -1,3 +1,7 @@
+v1.1.0 (Aug 19 2020)
+====================
+- Use checksums to prevent unnecessary uploads and downloads for unchanged assets
```

Comment:
@tanema Anything else worth highlighting here, e.g. #739 ?

Reply:
Definitely add `Strict theme ID handling to prevent accidental live theme alterations #739`

Also, should we include `Using checksums sourced from API rather than local filesystem in file watcher to optimize startup and fix changes made while the watch command was not running`

Reply:
👍 Updated